Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3 as base
```

Comment:
It would definitely be worth adding the expected build & run commands to your Readme so that they're recorded for the future!